schema/caveschema.xsd

```diff
@@ -1016,28 +1016,47 @@
     </xs:complexType>
   </xs:element>
 
-  <!-- Simple Types (string validation) -->
-  <!-- TODO: better validation here so regex isn't overly complicated in read_xml (31) -->
+
+  <!-- file (xs:string)
+    No validation done
+  -->
   <xs:simpleType name="file">
     <xs:annotation>
       <xs:documentation>A filename type</xs:documentation>
     </xs:annotation>
     <xs:restriction base="xs:string" />
   </xs:simpleType>
+
+  <!-- vector (xs:string)
+    Match "([float], [float], [float],)"
```

Comment:
It doesn't look like you're matching the trailing comma here on the last float. Is that on purpose? You can add a `,?` in your regex if you wanna optionally match that comma

Reply:
Haven't come across it before but I see no reason that would matter, added it!